tests/performance/templates/node-exporter-service.yaml

```diff
@@ -0,0 +1,16 @@
+{{ $count := (.Values.vmCount | int) }}
+{{- range until $count }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: svc-node-exporter-{{ $.Release.Name }}-{{ . }}
+  namespace: {{ $.Release.Namespace }}
+spec:
+  selector:
+    vm.kubevirt.io/name: {{ $.Release.Name }}-{{ . }}
+  ports:
+    - protocol: TCP
+      port: 9100
+      targetPort: 9100
+{{end}}
```

Comment:
if we use Service Monitor, then we don't need so many services. We can only create one with the label kubevirt.io=virt-launcher.